### PR TITLE
SDA-8716 | Fix: provide warning message when no OIDC config is found

### PR DESCRIPTION
**Pull request by willkutler**
ref: https://issues.redhat.com/browse/SDA-8716?filter=-1

### CODE REVIEW

File: cmd/create/cluster/cmd.go [hunk 1]
```diff
@@ -2368,10 +2368,8 @@ func handleOidcConfigOptions(r *rosa.Runtime, cmd *cobra.Command, isSTS bool, is
 		}
 	}
 	if oidcConfigId == "" {
-		if !isHostedCP {
-			return nil
```
Comment (gdbranco) [inline cmd/create/cluster/cmd.go:2372]: The only thing is that I feel like this message should show only for this one

If the cluster is a hosted CP cluster and the user specified args.classicOidcConfig it doesn't mean that an oidc configuration was not found but that the check was skipped, so it means the customer doesn't even want to check if there are any.

**Comment by willkutler** [comment]
@gdbranco thanks for the review - does this look good now?

**Comment by gdbranco** [comment]
> @gdbranco thanks for the review - does this look good now?

Would you mind adding a second check as a counter part to the args.classicOidcConfig?
Something similar to
```
if args.classicOidcConfig {
return nil
} else {
warn("No OIDC Configuration found;  Please create an OIDC Configuration through `rosa create oidc-configuration`")
os.Exit(1)
}
```
@pvasant mentioned would be better for UX

**Pull request merged**